libraries/AP_Baro/AP_Baro.cpp

```diff
@@ -232,6 +234,25 @@ const AP_Param::GroupInfo AP_Baro::var_info[] = {
     AP_GROUPINFO("_FIELD_ELV", 22, AP_Baro, _field_elevation, 0),
 #endif
 #endif
+
+#if APM_BUILD_COPTER_OR_HELI || APM_BUILD_TYPE(APM_BUILD_ArduPlane)
+    // @Param: _ALTERR_MAX
+    // @DisplayName: Maximum altitude error
```

Comment:
Ideally the parameter name should "Altitude error max" with the "Maximum" at the end.

Reply:
done